src/static/index.html

```diff
@@ -19,7 +19,7 @@
     <link rel="stylesheet" type="text/css" href="https://igv.org/web/release/1.0.1/igv-1.0.1.css">
     <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
     <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.2/jquery-ui.min.js"></script>
-    <script type="text/javascript" src="https://igv.org/web/release/1.0.1/igv-1.0.1.js"></script>
+    <script type="text/javascript" src="assets/js/igv-1.0.1.js"></script>
```

Comment:
I had to pull IGV locally so I could tweak it. See the second commit in this PR to see what I changed.